scrapy/mail.py

```diff
@@ -112,6 +112,7 @@ def _sent_failed(self, failure, to, cc, subject, nattachs):
     def _sendmail(self, to_addrs, msg):
         # Import twisted.mail here because it is not available in python3
         from twisted.mail.smtp import ESMTPSenderFactory
+        from twisted.internet import reactor
```

Comment:
You can reorder these so that they are sorted :)

Reply:
Okay